schema/2024-09/Form.json

```
@@ -188,36 +243,165 @@
         "username": {
           "description": "Username of the editor.",
           "type": "string"
+        },
+        "isSuperuser": {
+          "description": "User is a super-user.",
+          "type": "boolean"
+        },
+        "isSupportUser": {
+          "description": "User is a support-user.",
+          "type": "boolean"
+        },
+        "hasAccessForCourseEvaluation": {
+          "description": "User has access to course evaluation form type.",
+          "type": "boolean"
+        }
+      }
+    },
+    "netgroup": {
+      "description": "Netgroup editor.",
+      "type": "object",
+      "required": ["netgroupId", "name"],
+      "properties": {
+        "netgroupId": {
+          "description": "Identifier for the netgroup.",
+          "type": "integer"
+        },
+        "name": {
+          "description": "Name of the netgroup.",
+          "type": "string"
         }
       }
     },
     "element": {
       "description": "Elements of the skjema.",
       "type": "object",
       "additionalProperties": false,
-      "required": ["sequence", "mandatory"],
+      "required": ["sequence", "elementId"],
       "properties": {
+        "elementId": {
+          "description": "Internal identifier for this element.",
+          "type": ["integer", "null"]
```

```suggestion
          "type": ["integer"]
```
ElementId kan aldri være null

---

Dessverre kan den være null. Foreløpig ser det ut som det kun skjer dersom type er `PAGE_BREAK`:

```json
{
  "text": null,
  "altText": null,
  "sequence": null,
  "elementId": null,
  "dateFormat": null,
  "description": null,
  "elementType": "PAGE_BREAK",
  "subElements": [],
  "answerOptions": [],
  "isAnswerHashed": false,
  "externalElementId": null,
  "nationalIdNumberType": null,
  "useForRedirectToForm": false
}
```